### PR TITLE
Update new Exchange admin center location in mya-disable-insights.md

### PR DESCRIPTION
**Pull request by dariomws**
Closes https://github.com/MicrosoftDocs/Viva/issues/430.

Closes https://github.com/MicrosoftDocs/Viva/issues/478.

### CODE REVIEW

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@madehmer 
PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by tiburd** [comment]
@dariomws 
if you can share a screenshot of the changed menu item, here or via email, and then @ mention me, I can approve and merge this PR.
Thanks!

**Comment by dariomws** [comment]
@tiburd, sure:

image

Screenshot of the new Exchange Admin Center.

File: Viva/insights/personal/setup/mya-disable-insights.md [hunk 1]
```diff
@@ -24,8 +24,8 @@ You can use the following steps to disable the Microsoft Viva Insights Outlook a
 > If you're a targeted release customer, you might see a new admin experience. To learn how to disable the add-in through this new experience, refer to [Using the new admin experience](../../advanced/setup-maint/new-admin-experience.md#manage-default-settings). 
 
 1. Sign in to the [Microsoft 365 admin center](https://admin.microsoft.com/adminportal).
-2. In the left navigation pane, select **Exchange**. This opens the dashboard of the Exchange admin center.  
-3. In the dashboard, select **add-ins**.
+2. In the left navigation pane, select **Exchange**. This opens the Exchange admin center.  
+3. Select **Organization** > **add-ins**.
 4. In the list of add-ins, select **Microsoft Viva Insights**, and then select the **Edit** (pencil) icon. This opens the **Edit add-in settings** dialog box.
```
Comment (tiburd) [inline Viva/insights/personal/setup/mya-disable-insights.md:29]:
```suggestion
4. In the list of add-ins, select **Viva Insights**, and then select the **Edit** (pencil) icon. This opens the **Edit add-in settings** dialog box.
```

**Comment by tiburd** [comment]
@dariomws 
Thank you! I added a couple edits based on the screenshot.

**Pull request merged**